aries-site/src/layouts/content/InPageNavigation.js

```diff
@@ -123,10 +144,20 @@ export const InPageNavigation = ({ headings }) => {
                         ? undefined
                         : { left: theme.global.borderSize.small }
                     }
+                    direction="row"
+                    align="top"
+                    gap="small"
                   >
                     <Text color="text-strong" size="small" weight="normal">
                       {headingTitle}
                     </Text>
+                    {newUpdate && (
+                      <Box justify="top">
+                        <Text>
+                          <StatusGoodSmall size="10px" color="#117B82" />
```

Comment:
Best practice note, we always want to use t-shirt sizes for sizing and theme tokens for colors. Can this be updated to `size="small"` or `size="xsmall"` (depending on what is appropriate) and the theme's color token (Maybe `teal`?)

Reply:
Yeah totally! I chatted with Jessica about [this one](https://github.com/grommet/hpe-design-system/pull/3502#discussion_r1289127971) a couple of comments up, but the grommet icons don't take the hpe theme colors so I just grabbed the hex from the figma. In terms of the size, unfortunately the "xsmall" was way too big so we decided to result to px for this one :/

Reply:
Just want to note that the grommet icon can use the hpe theme colors it just can't take the format where we specify that it should use the dark version of the color.